src/vs/workbench/browser/parts/editor/editorGroupView.ts

```diff
@@ -869,8 +870,8 @@ export class EditorGroupView extends Themable implements IEditorGroupView {
 		}
 	}
 
-	invokeWithinContext<T>(fn: (accessor: ServicesAccessor) => T): T {
-		return this.scopedInstantiationService.invokeFunction(fn);
+	get scopedContextKeyService(): IContextKeyService {
```

Comment:
Just :lipstick: but I prefer `get` and the associated `private` field to be close to each other, so can you move it up right below `_scopedContextKeyService`?